portfolio/src/main/webapp/index.html

```diff
@@ -38,7 +38,32 @@ <h3> My information </h3>
       </button>
       <p id = "rand-element"></p>
     
+      <form action="/form-handler" method="POST">
```

Comment:
Assuming the scenario works end to end... I think this commit doesn't include new servlet vile from server side.